packages/styles/scss/components/content-block-horizontal/_content-block-horizontal.scss

```diff
@@ -9,12 +9,22 @@
 @import '../content-item-horizontal/content-item-horizontal';
 
 @mixin content-block-horizontal {
+  :host(#{$dds-prefix}-content-block-horizontal),
+  .#{$prefix}--content-block-horizontal {
+    /* stylelint-disable-next-line selector-type-no-unknown */
+    dds-hr,
+    .bx--content-block .bx--hr {
+      @include carbon--breakpoint('lg') {
+        margin-right: -9rem;
```

Comment:
Where is this being applied? And why would such a large value be needed on a margin?

Reply:
This has been removed 👍